### ISSUE TITLE
docs: Add a section in the documentation displaying versions of the frameworks and extension use. 

### ISSUE THREAD

**Issue by Amulya-coder** [anitab-org/bridge-in-tech-backend#223]
## Description
In the readme add a section that shows the framework version and extensions used in the project.

## Acceptance Criteria
Add the versions and extensions used in bridge-in-tech

### Update [Required]
- [ ] Currently used versions,  extensions should be added

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

**Comment by Amulya-coder** [comment]
@mtreacy002  I think this is a valid issue for `first timers only` can you please verify.

**Comment by mtreacy002** [comment]
@Amulya-coder , can you please explain which specific framework and extension you want the contributors to add here? Do you mean the dependencies we use in the requirements.txt? If so, do we still need to state them on the Readme when contributors can see them from the requirements.txt?

**Comment by Amulya-coder** [comment]
Yes, @mtreacy002 the flask extensions and framework that are mentioned in the requirements.txt and this helps the new contributors to learn more about the project work if we defining the extensions, framework, Language used for developing the project in the readme so that it is easily noticed by the contributors.

**Comment by Rahulm2310** [comment]
@Amulya-coder Thanks for opening this but I agree here with @mtreacy002. Also, github UI shows the languages used in the project. 
![image](https://user-images.githubusercontent.com/54268438/112351994-68030480-8cf0-11eb-8874-dc846389371a.png)

Furthermore, we also have tags for project frameworks and languages in the about section.
![image](https://user-images.githubusercontent.com/54268438/112352226-9b459380-8cf0-11eb-9127-4c7f07e394dd.png)

So, I don't think there is need for adding redundant details again to the readme.

**Comment by Amulya-coder** [comment]
@Rahulm2310 Ahh... I know that there are default tags provided in the Github but it's a "good practice" to add it in the documentation also. This is used in the [mentorship Backened](https://github.com/anitab-org/mentorship-backend#documentation) too you can check :) .

**Comment by matsy** [comment]
@mtreacy002 Is this issue open? Is someone working on this?
